src/wasm-s-parser.h

```diff
@@ -450,8 +450,18 @@ class SExpressionWasmBuilder {
             if (op[2] == 'p') return makeBinary(s, BinaryOp::CopySign, type);
             if (op[2] == 'n') {
               if (op[3] == 'v') {
-                if (op[8] == 's') return makeUnary(s, op[11] == '3' ? UnaryOp::ConvertSInt32 : UnaryOp::ConvertSInt64, type);
-                if (op[8] == 'u') return makeUnary(s, op[11] == '3' ? UnaryOp::ConvertUInt32 : UnaryOp::ConvertUInt64, type);
+                if (op[8] == 's') {
+                  if (op[10] == 'i')
```

Comment:
Please use `{,}` when the body of the `if` is on a separate line, for all these.